### PR TITLE
Fix clang-format comment indents on Windows for winml/

### PR DESCRIPTION
**Pull request by justinchuby**
On Windows, clang-format has a bug when AlignTrailingComments.Kind is set to `Leave` (https://clang.llvm.org/docs/ClangFormatStyleOptions.html#aligntrailingcomments), where it will keep adding indentation to comments after each formatting runs.

This PR changes to always align comments so we do not hit the bug.

As a consequence of the options change we need to reformat some of the files. Note that this option is aligned with the rest of the repository.

### CODE REVIEW

File: winml/test/api/LearningModelBindingAPITest.cpp [hunk 1]
```diff
@@ -367,60 +367,60 @@ static void VerifyInvalidBindExceptions() {
   auto ensureWinmlSizeMismatch = std::bind(matchException, std::placeholders::_1, WINML_ERR_SIZE_MISMATCH);
   auto ensureWinmlInvalidBinding = std::bind(matchException, std::placeholders::_1, WINML_ERR_INVALID_BINDING);
 
-    /*
+  /*
         Verify tensor bindings throw correct bind exceptions
     */
```
Comment (fdwr) [inline winml/test/api/LearningModelBindingAPITest.cpp:372]: [nit] Is the opening/closing assignment deliberate?

```
  /*
        Verify tensor bindings throw correct bind exceptions
    */
```

--> ?

```
  /*
        Verify tensor bindings throw correct bind exceptions
  */
```

**Review by fdwr — APPROVED** [review]
So this fixes the issue Pat observed then 👍. TY. Added one minor comment, but otherwise looks fine to me.

**Review by snnn — APPROVED** [review]

File: winml/test/api/LearningModelBindingAPITest.cpp (hunk 1 above)
Comment (justinchuby) [reply winml/test/api/LearningModelBindingAPITest.cpp:372]: Thanks. I will create a separate manual adjustment for this. I don’t think it’s intentional but the formatter wasn’t smart enough to fix it.

Comment (justinchuby) [reply winml/test/api/LearningModelBindingAPITest.cpp:372]: https://github.com/microsoft/onnxruntime/pull/17163

**Pull request merged**